### ISSUE TITLE
Some entries in the Events Directory are duplicated for the Lab Officer user

### ISSUE THREAD

**Issue by roxanamlendea** [hzi-braunschweig/SORMAS-Project#9082]
### Bug Description
On certain testing instances (my local environment, test-de2.sormas.netzlink.com and test1401.sormas-oegd.de - version 1.70.3), on the Events Directory, Events are duplicated.

Notes:
1. This is not reproducible on @adinaflorea9 's local environment and on test-de1.sormas.netzlink.com.
2. I tried to figure out why these Events are displayed twice, but unfortunately I couldn't figure out why.

### Steps to Reproduce

1. Log in with Lab Officer user;
2. Navigate to the Events Directory;
3. Inspect the entries;

### Expected Behavior
Events should not be displayed twice.

### Screenshots
local environment:
![image](https://user-images.githubusercontent.com/89522688/166886434-14f9e584-7e24-44ec-a90b-b8e445aea811.png)

test1401.sormas-oegd.de - version 1.70.3
![image](https://user-images.githubusercontent.com/89522688/166886331-7933e5bc-2cba-4a26-8e6f-985b7ecc56e4.png)

test-de2.sormas.netzlink.com
![image](https://user-images.githubusercontent.com/89522688/166886263-be52a35d-8b2a-4c1a-8c6c-5a420992a698.png)

### System Details

* Device: Windows 10
* SORMAS version: Sormas 1.71.0-SNAPSHOT (c69ead01) and 1.70.3
* Android version/Browser: Chrome
* Server URL: my local environment, test-de2.sormas.netzlink.com and test1401.sormas-oegd.de
* User Role: Lab Officer

### Additional Information

- `EventFacadeEjb.getIndexList` already has `DISTINCT` set, but that does not seem to help. Probably the jurisdiction check or the user filter duplicates data for some reason.

**Comment by StefanKock** [comment]
The query for lab user does give some options for duplicated rows:, Might be related to multiple `EventParticipant` (`eventperso7_`) or `Case` `Contact`s (`contacts9_`)

<details>
<summary>lab user SELECT query for getIndexList</summary>

```sql
SELECT DISTINCT event0_.id AS col_0_0_,
                event0_.uuid AS col_1_0_,
                event0_.externalId AS col_2_0_,
                event0_.externalToken AS col_3_0_,
                event0_.internalToken AS col_4_0_,
                event0_.eventStatus AS col_5_0_,
                event0_.riskLevel AS col_6_0_,
                event0_.specificRisk AS col_7_0_,
                event0_.eventInvestigationStatus AS col_8_0_,
                event0_.eventManagementStatus AS col_9_0_,
                event0_.disease AS col_10_0_,
                event0_.diseaseVariant AS col_11_0_,
                event0_.diseaseDetails AS col_12_0_,
                event0_.startDate AS col_13_0_,
                event0_.endDate AS col_14_0_,
                event0_.evolutionDate AS col_15_0_,
                event0_.eventTitle AS col_16_0_,
                region2_.uuid AS col_17_0_,
                region2_.name AS col_18_0_,
                district3_.uuid AS col_19_0_,
                district3_.name AS col_20_0_,
                community4_.uuid AS col_21_0_,
                community4_.name AS col_22_0_,
                location1_.city AS col_23_0_,
                location1_.street AS col_24_0_,
                location1_.houseNumber AS col_25_0_,
                location1_.additionalInformation AS col_26_0_,
                event0_.srcType AS col_27_0_,
                event0_.srcFirstName AS col_28_0_,
                event0_.srcLastName AS col_29_0_,
                event0_.srcTelNo AS col_30_0_,
                event0_.srcMediaWebsite AS col_31_0_,
                event0_.srcMediaName AS col_32_0_,
                event0_.reportDateTime AS col_33_0_,
                user5_.uuid AS col_34_0_,
                user5_.firstName AS col_35_0_,
                user5_.lastName AS col_36_0_,
                user6_.uuid AS col_37_0_,
                user6_.firstName AS col_38_0_,
                user6_.lastName AS col_39_0_,
                CASE
                    WHEN (event0_.reportingUser_id IS NOT NULL)
                         AND event0_.reportingUser_id=63
                         OR (event0_.responsibleUser_id IS NOT NULL)
                         AND event0_.responsibleUser_id=63
                         OR EXISTS
                           (SELECT sample10_.id
                            FROM samples sample10_
                            LEFT OUTER JOIN EventParticipant eventparti11_ ON sample10_.associatedEventParticipant_id=eventparti11_.id
                            LEFT OUTER JOIN users user12_ ON sample10_.reportingUser_id=user12_.id
                            LEFT OUTER JOIN Facility facility13_ ON sample10_.lab_id=facility13_.id
                            WHERE eventparti11_.id=eventperso7_.id
                              AND ((user12_.id IS NOT NULL)
                                   AND user12_.id=63
                                   OR facility13_.id=62)) THEN TRUE
                    ELSE FALSE
                END AS col_40_0_,
                event0_.changeDate AS col_41_0_,
                event0_.eventIdentificationSource AS col_42_0_
FROM EVENTS event0_
LEFT OUTER JOIN LOCATION location1_ ON event0_.eventLocation_id=location1_.id
LEFT OUTER JOIN Region region2_ ON location1_.region_id=region2_.id
LEFT OUTER JOIN District district3_ ON location1_.district_id=district3_.id
LEFT OUTER JOIN Community community4_ ON location1_.community_id=community4_.id
LEFT OUTER JOIN users user5_ ON event0_.reportingUser_id=user5_.id
LEFT OUTER JOIN users user6_ ON event0_.responsibleUser_id=user6_.id
LEFT OUTER JOIN EventParticipant eventperso7_ ON event0_.id=eventperso7_.event_id
LEFT OUTER JOIN cases case8_ ON eventperso7_.resultingCase_id=case8_.id
LEFT OUTER JOIN contact contacts9_ ON case8_.id=contacts9_.caze_id
WHERE (EXISTS
         (SELECT sample14_.id
          FROM samples sample14_
          LEFT OUTER JOIN EventParticipant eventparti15_ ON sample14_.associatedEventParticipant_id=eventparti15_.id
          LEFT OUTER JOIN users user16_ ON sample14_.reportingUser_id=user16_.id
          LEFT OUTER JOIN Facility facility17_ ON sample14_.lab_id=facility17_.id
          WHERE eventparti15_.id=eventperso7_.id
            AND ((user16_.id IS NOT NULL)
                 AND user16_.id=63
                 OR facility17_.id=62) )
       OR EXISTS
         (SELECT sample18_.id
          FROM samples sample18_
          LEFT OUTER JOIN cases case19_ ON sample18_.associatedCase_id=case19_.id
          LEFT OUTER JOIN Facility facility20_ ON sample18_.lab_id=facility20_.id
          WHERE case19_.id=case8_.id
            AND facility20_.id=? )
       OR contacts9_.reportingUser_id=?
       OR contacts9_.contactOfficer_id=?
       OR EXISTS
         (SELECT sample21_.id
          FROM samples sample21_
          LEFT OUTER JOIN contact contact22_ ON sample21_.associatedContact_id=contact22_.id
          LEFT OUTER JOIN Facility facility23_ ON sample21_.lab_id=facility23_.id
          WHERE contact22_.id=contacts9_.id
            AND facility23_.id=? )
       OR case8_.sharedToCountry=TRUE
       OR case8_.reportingUser_id=63
       OR case8_.surveillanceOfficer_id=63
       OR case8_.caseOfficer_id=63
       OR event0_.reportingUser_id=?
       OR event0_.responsibleUser_id=?)
  AND event0_.deleted=?
ORDER BY event0_.changeDate DESC
LIMIT ?
```

</details>

**Comment by StefanKock** [comment]
The problem is not reproducable on 1.72.0-SNAPSHOT anymore fc5ffb507ceddb296cc4fdb8235762b2a0dcf86c , but still on 1.70.x

We should check on which version it got fixed, and provide the added unit test either way because we don't have such a test yet.

**Comment by roxanamlendea** [comment]
I also checked on my local machine (1.72.0-SNAPSHOT) (f5fc041) and on test-de2.sormas.netzlink.com , and it is not reproducible anymore. 
It is reproducible on 1.71.0 on test1401.sormas-oegd.de
![image](https://user-images.githubusercontent.com/89522688/168033124-633f07b7-1962-42b9-8d61-8bdb054d627c.png)

**Comment by StefanKock** [comment]
Bug was solved in 1.72.0, probably by #8747 without noticing it.
